### PR TITLE
[DASH-2295] Add CNAME input

### PR DESCRIPTION
**Pull request by tomsonpl**
please review
screen shot 2016-10-14 at 18 56 20

### CODE REVIEW

File: src/apps/Hosting/HostingDialog.js [hunk 1]
```diff
@@ -11,7 +10,7 @@ import SessionStore from '../Session/SessionStore';
 
 import { TextField, Toggle } from 'material-ui';
 import { Dialog, Show, Notification } from '../../common';
-import HostingDialogDomainTable from './HostingDialogDomainTable';
+// import HostingDialogDomainTable from './HostingDialogDomainTable';
```
Comment (patrykkopycinski) [inline src/apps/Hosting/HostingDialog.js]: delete it

File: src/apps/Hosting/HostingListItemLinks.js [hunk 2]
```diff
@@ -46,7 +46,7 @@ const HostingListItemLinks = ({ isVisible, params, domains }) => {
   };
   const renderHostingLinks = () => {
     const { instanceName } = params;
-    const domainsToDisplay = _.without(domains, 'default');
+    const domainsToDisplay = _.reject(domains, { value: 'default' });
```
Comment (patrykkopycinski) [inline src/apps/Hosting/HostingListItemLinks.js]: why this component has to know anything about `CNAME`?

File: src/apps/Hosting/HostingList.js [hunk 3]
```diff
@@ -49,6 +50,9 @@ const HostingList = React.createClass({
 
   renderItem(item) {
     const { checkItem } = this.props;
+    const cname = _.reject(item.domains, { value: 'default' })[0];
```
Comment (patrykkopycinski) [inline src/apps/Hosting/HostingList.js]: why this component has to know anything about `CNAME`?

Comment (tomsonpl) [inline src/apps/Hosting/HostingList.js]: hmm I needed to add cname to an item. I tried to do it in the dialog, but then I was getting a red screen on the list ;/ could you give me any hint?

**Pull request merged**